rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -215,6 +217,11 @@ create_node(
       RMW_SET_ERROR_MSG("failed to allocate memory for 'perm_fn'");
       goto fail;
     }
+    logging_fn = rcutils_join_path(srp, "logging.xml", allocator);
```

Comment:
@mikaelarguedas if this file wasn't located alongside the other security files, we would probably need to modify RCL to find it for us, which limits the benefit of doing any of this parsing at the RMW level (since it requires an RCL API update anyway).

Reply:
If this file is meant to be ROS specific and contain only ROS concepts. Maybe it makes more sense to parse it in RCL (where the ROS concepts live). This would allow to avoid any duplication of parsing code and each RMW implementation (DDS or not) can use the parsed information as it sees fit.

This stands regardless of where the file is located. However, having that logic in RCL would enable more easily support for the case the file is at another location